dev-python/unasync/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>python@gentoo.org</email>
+	</maintainer>
+	<!-- maintainer-needed -->
```

Comment:
Probably better to replace the python project with you + proxy maint (if not consulted otherwise with the python team).

Reply:
Added myself as proxy maint and removed the python project as I haven't consulted anyone in the python team.